stateful-workload-filestore/indexInfo.html

```diff
@@ -0,0 +1,64 @@
+<!--
+ Copyright 2022 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ -->
+
+<html>
+
+<head>
+    <title>Stateful Workload Tutorial</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
```

Comment:
jQuery is bulky, do we really need it for a 10-line ES script? It's not even leveraged fully since you're calling things like `document.getElementById()` instead of `$(#)` 😄

Reply:
I can help refactor this! :)

Reply:
Ideally, we would avoid jQuery to minimize the amount of dependencies we're using (vulnerability surface).
But it looks like jQuery isn't bulky at all (only `33.4kb`). :)
![Screen Shot 2022-07-04 at 3 54 48 PM](https://user-images.githubusercontent.com/10292865/177210983-829845f0-d9cf-46d0-82a4-0119aa3e1cef.png)

Nonetheless, I explored the option of replacing jQuery with just vanilla JavaScript (e.g., using `fetch` instead of [`load`](https://api.jquery.com/load/) and parsing the `indexData.html` file with vanilla JS), and the effort required is 1+ hour. :(

@charlieyu1996 
I've made some tweaks to the JavaScript here: https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/pull/325
My pull-request is embracing jQuery more fully (e.g., replacing `document.getElementById`) and includes other _minor_ tweaks.
Feel free to merge or pick-and-choose parts of it.

Reply:
Thanks Olivier and Nim! I have merged the changes.